erp/export/export.py

```diff
@@ -0,0 +1,11 @@
+import csv
+from dataclasses import asdict
+from typing import List
+
+
+def export_to_csv(file, headers, mapped_data: List):
+    csv_writer = csv.DictWriter(file, fieldnames=headers)
```

Comment:
We should probably double-check the defaults (separator, quoting) used by DictWriter to ensure it matches Etalab's expectations.